src/nvim/path.c

```diff
@@ -2365,6 +2365,11 @@ int append_path(char *path, const char *to_append, size_t max_len)
 ///
 /// @return FAIL for failure, OK for success.
 static int path_to_absolute(const char *fname, char *buf, size_t len, int force)
+#ifdef MSWIN
+{
+  return _fullpath(buf, fname, len) != NULL;
```

Comment:
assuming the logic below is actually valuable (its using various Nvim internal state/options), can we force `/` slashes, do the logic, then force `\` slashes only in the `f_xx` vimscript functions?

Reply:
> can we force `/` slashes and do the logic?

If you mean calling `forward_slash()` on the `buf` string, then I did that just now.  As for doing the logic below, I don't think it's valuable.  As far as I can see, it only deals with local/temp variables which are only used in the purpose of setting `buf` to the absolute path of `fname`.

Reply:
Maybe `path_to_absolute()' should be moved to `os/` specific module?